lib/vtls/schannel.h

```diff
@@ -61,16 +71,73 @@ CURLcode Curl_verify_certificate(struct Curl_easy *data,
 
 #ifdef __MINGW32__
 #include <_mingw.h>
-#ifdef __MINGW64_VERSION_MAJOR
+#if defined(__MINGW64_VERSION_MAJOR) && defined(_WIN32_WCE)
```

Comment:
What's this for? Why restrict this to Windows CE?

Reply:
Letting SChannel handle certificate verification (rather than duplicating work -- and potentially introducing bugs). The "do it in curl" option was originally to handle Windows CE, then somehow down the line got enabled for everyone.

More discussion here: https://github.com/curl/curl/pull/7784#discussion_r717549197

Reply:
Hmm, this looks to me like it shouldn't be disabled for non-CE: https://github.com/curl/curl/blob/13d7aec2a3794664d451aa27bdbad7a45166db91/lib/vtls/schannel.c#L1050

Reply:
SChannel doesn't use CACerts files (even the [curl docs state that](https://curl.se/docs/sslcerts.html#certificate-verification-with-schannel-and-secure-transport)). That's an odd bit of code, for sure.

Reply:
Sseems lik the Cirrus CI tests are failing because of that change:
```
curl: (4) schannel: CA cert support not built in
```

Reply:
schannel is supposed to have manual verify support, it is needed for CA certificate bundles. so no it's not for windows ce anymore

Reply:
Ok, I’ll modify the change on Monday to use manual verification if needed and use the built in SChannel verifier otherwise. 

Then, after this PR is merged I’ll make another PR with a DISABLE_X define to optionally reduce the code-bloat of this niche feature.

Reply:
Ok, I backed-out this change. I'll make a separate PR later that will optionally exclude these niche features (and reduce compiled code-size).